### PR TITLE
Update `Global Moderation Team`

### PR DESCRIPTION
**Pull request by Blooshing**
Been a while since I've contributed, along with `osu-wiki translate-groups` wasn't working for whatever reason (Error was ENOENT), regardless let me know if there are any issues. 

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/Global_Moderation_Team/de.md [hunk 1]
```diff
@@ -131,6 +127,7 @@ Die [Seite für das globale Moderationsteam](https://osu.ppy.sh/groups/4) listet
 | ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927) | Philippinisch | Chat-Moderation |
 | ::{ flag=TH }:: [Trigonoculus](https://osu.ppy.sh/users/7627013) | Thailändisch | Chat-Moderation |
 | ::{ flag=ES }:: [Trosk-](https://osu.ppy.sh/users/3469385) | Spanisch | Technischer Support |
+| ::{ flag = NL }:: [Uni](https://osu.ppy.sh/users/617106) | Hollandisch | Chat-Moderation |
```
Comment (Walavouchey) [inline wiki/People/Global_Moderation_Team/de.md:130]:
```suggestion
| ::{ flag=NL }:: [Uni](https://osu.ppy.sh/users/617106) | Hollandisch | Chat-Moderation |
```

File: wiki/People/Global_Moderation_Team/pl.md [hunk 2]
```diff
@@ -86,23 +85,21 @@ Lista wszystkich moderatorów globalnych jest dostępna na [stronie grupy](https
 | ::{ flag=FI }:: [Laurakko](https://osu.ppy.sh/users/7253731) | Fiński | Moderowanie czatu |
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Portugalski, hiszpański | Moderowanie czatu, organizowanie turniejów |
 | ::{ flag=DE }:: [Loctav](https://osu.ppy.sh/users/71366) | Niemiecki | Moderowanie społeczności twórców i moderów beatmap |
-| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Kantoński, chiński | Organizowanie turniejów |
+
```
Comment (Walavouchey) [inline wiki/People/Global_Moderation_Team/pl.md:88]:
```suggestion
```

**Review by Walavouchey — COMMENTED** [review]
Uni hasn't been added in all languages... ?

File: wiki/People/Global_Moderation_Team/de.md (hunk 1 above)
Comment (Blooshing) [reply wiki/People/Global_Moderation_Team/de.md:130]: osu-wiki translate-groups wasn't working on my laptop, going to try it on a different computer to in a little bit.

**Comment by Blooshing** [comment]
should be good now, methinks

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**